### PR TITLE
Props and drivers

### PR DESCRIPTION
**Pull request by clicman**
We need to avoid driver file name in code. Need to move it to porps #28

### CODE REVIEW

File: src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java [hunk 1]
```diff
@@ -82,93 +85,64 @@
...
-                    } else {
-                        LOG.warn("The value of property 'webdriver.drivers.path is not specified."
-                                + " Try to get {} driver from system PATH'", WEBDRIVER_BROWSER_NAME);
-                    }
-                    capabilities.setBrowserName("internet explorer");
+                }
+                break;
+            case BrowserType.IE:
+            case BrowserType.IE_HTA:
+            case BrowserType.IEXPLORE:
```
Comment (warorc) [inline src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java]: what's the difference between all of this types

File: src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java [hunk 2]
```diff
@@ -82,93 +85,64 @@
...
+            default:
+                throw new UnsupportedBrowserException("'" + WEBDRIVER_BROWSER_NAME + "' is not supported yet");
+        }
+        if (!WEBDRIVER_URL.isEmpty()) {
             try {
                 URL remoteUrl = new URL(WEBDRIVER_URL);
                 setWebDriver(new RemoteWebDriver(remoteUrl, capabilities));
             } catch (MalformedURLException e) {
-                LOG.error("Could not parse remote url. Check 'webdriver.url' property");
+                LOG.error("Could not parse remote url. Check 'webdriver.url' property", e);
```
Comment (kosteman) [inline src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java]: Maybe error throwing is enough?

File: src/main/resources/config/application.properties-sample [hunk 3]
```diff
@@ -42,7 +42,7 @@ webdriver.create.attempts = 3
 webdriver.proxy.enabled = false
 
 #optional. If path is not specified webdrivers will be searched in system PATH
-webdriver.drivers.path = src/test/resources/webdrivers/
+webdriver.drivers.path = src/test/resources/webdrivers/chromedriver.exe
```
Comment (kosteman) [inline src/main/resources/config/application.properties-sample]: So now if we need to switch between browsers we should change two properties - webdriver.drivers.path and webdriver.browser.name, right?

Comment (clicman) [inline src/main/resources/config/application.properties-sample]: Yes.

Comment (kosteman) [inline src/main/resources/config/application.properties-sample]: Can we simplify it?

Comment (clicman) [inline src/main/resources/config/application.properties-sample]: No, we can't. We don't know what filename will into user environment.

File: src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java (hunk 1 above)
Comment (kosteman) [inline src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java]: webdriver.browser.name=IE_HTA or
webdriver.browser.name=IE HTA?

File: src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java (hunk 2 above)
Comment (clicman) [inline src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java]: No. We always should pass exception into log if it error

File: src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java (hunk 1 above)
Comment (clicman) [inline src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java]: public interface BrowserType {
    String FIREFOX = "firefox";
    String FIREFOX_PROXY = "firefoxproxy";
    String FIREFOX_CHROME = "firefoxchrome";
    String GOOGLECHROME = "googlechrome";
    String SAFARI = "safari";
    /** @deprecated */
    @Deprecated
    String OPERA = "opera";
    String OPERA_BLINK = "operablink";
    String EDGE = "MicrosoftEdge";
    String IEXPLORE = "iexplore";
    String IEXPLORE_PROXY = "iexploreproxy";
    String SAFARI_PROXY = "safariproxy";
    String CHROME = "chrome";
    String KONQUEROR = "konqueror";
    String MOCK = "mock";
    String IE_HTA = "iehta";
    String ANDROID = "android";
    String HTMLUNIT = "htmlunit";
    String IE = "internet explorer";
    String IPHONE = "iPhone";
    String IPAD = "iPad";
    String PHANTOMJS = "phantomjs";
}

**Pull request merged**